### DISCUSSION TITLE
Can`t use installed depth model due to "Incorrect model type selected for prompt to image tasks."

### DISCUSSION THREAD

**Discussion by SebastianVRLab** [carson-katri/dream-textures#630] [Q&A]
Can`t Generate texture due to the error mentioned above.

I downloded, un-zipped and installed the current version of dream_textures-windows-cuda for Nvidia GPU.
I installied the stable diffusion and depth model stabilityai/stable-diffusion-2-depth.

Wiped all Addon Data, reinstalled, but withouth any improvement.
Tried with Blender 3.4, 3.5

I was able to generate a Texture with Blender 3.3 for a few times, then the error mentioned above occured. When closing Blender and creating a fresh .blend file, the error initially does not occur, but eventually comes back.

Any advise how I might fix this?
Thanks a lot!
![2023-04-12_14h44_57](https://user-images.githubusercontent.com/114126172/231363276-31bd2f13-433a-488a-8ea4-36aa364126d6.png)

**Comment by carson-katri** [comment]
You need to use the panel in the viewport to use projection. You have the image editor panel open.
